src/store/Store.js

```diff
@@ -12,6 +12,9 @@ import uniqueID from '../uniqueid/uniqueID.js';
 const HINT_LINK = 'Learn more about using the Store API:' +
   ' https://github.com/HubSpot/general-store#using-the-store-api';
 
+const hasReduxDevTools = typeof window !== 'undefined' &&
```

Comment:
Is the redux stuff something that should only live in the development version?

If so, you can wrap all the logic  in `if (process.env.NODE_ENV !== 'production') {` and it'll be dead-code eliminated in the production version 

For example...
https://github.com/HubSpot/general-store/blob/100b9913300f5d7ce0dd5174d6a76485faff1795/src/store/Store.js#L106-L115

Reply:
I can see it being pretty useful in the prod build and helping debug / inspect state while deployed. But if you think otherwise I'm cool with adding this.

Reply:
I'm happy to defer to redux's behavior. Do the redux devtools work in prod?